packages/stark-ui/src/modules/table/components/column.component.html

```diff
@@ -39,4 +39,7 @@
 			*ngTemplateOutlet="columnTemplate; context: { $implicit: { rowData: rowItem, displayedValue: getDisplayedValue(rowItem) } }"
 		></ng-container>
 	</td>
+	<td mat-footer-cell *matFooterCellDef>
```

Comment:
Shoudln't this footer be added only if footerValue is not `undefined`?

Reply:
yes true